css/style.css

```
@@ -15,5 +14,178 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+.header {
+  display: flex;
+  justify-content: space-around;
+}
+#logo {
+  width: 100px;
+  height: 50px;
+}
+
+.navbar ul {
+  display: flex;
+  flex-direction: row;
+  list-style-type: none;
+}
+.navbar ul li {
+  padding: 10px;
+}
+
+.list-nav li a {
+  text-decoration: none;
+  color: black;
+  opacity: 0.6;
+}
+.list-nav li a:hover {
+  color: rgb(248, 92, 44);
+}
+.list-nav li a:active {
+  color: black;
+}
+#menu {
+  display: none;
+}
+.section-1 {
+  background-image: url("/img/first-background.jpg");
+  height: 730px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  margin: 0px;
```

```suggestion
  margin: 0;
```
In CSS when we use a value of 0 we usually leave out the units, so leave out `rem` or `px` for example